synapse/lib/stormtypes.py

```diff
@@ -803,6 +828,9 @@ def __init__(self, valu, path=None):
         StormType.__init__(self, path=path)
         self.valu = valu
 
+    def __len__(self):
+        return len(self.valu)
```

Comment:
This is the opposite behavior that we discussed. I thought we were going to have ``Prim.__len__(self)`` raise a StormRuntimeError, and make types which can produce useful lengths implement ``__len__``.

Reply:
this implementation allows us to leak attributeerror messages to the remote caller